articles/migrate/tutorial-migrate-vmware.md

```diff
@@ -241,9 +241,9 @@ After you've verified that the test migration works as expected, you can migrate
 4. A migration job starts for the VM. Track the job in Azure notifications.
 5. After the job finishes, you can view and manage the VM from the **Virtual Machines** page.
 
-## Complete the migration
+## Complete the Migration
 
-1. After the migration is done, right-click the VM > **Stop migration**. This stops replication for the on-premises machine, and cleans up replication state information for the VM.
+1. After the migration is done, right-click the VM > **Stop Replication**. This stops replication for the on-premises machine, and cleans up replication state information for the VM.
```

Comment:
Approved.

Reply:
Thank you David, updated the migration based on feedback.